Sources/MongoSwift/BSON/BSONEncoder.swift

```diff
@@ -5,12 +5,16 @@ import mongoc
 public class BSONEncoder {
 
     /// Enum representing the various strategies for encoding `Date`s.
+    ///
+    /// As per the BSON specification, the default strategy is to encode `Date`s as BSON datetime objects.
+    ///
+    /// See bsonspec.org for more information.
```

Comment:
these will show up more cnicely in the generated docs if we write them as "see alsos".
see [here](https://github.com/mongodb/mongo-swift-driver/blob/2610f57ac2c937a039b35e331086e225ea5f0cbf/Sources/MongoSwift/ReadPreference.swift#L6) for an example of this in code, and [here's](https://mongodb.github.io/mongo-swift-driver/Classes/ReadPreference.html) how it looks in the docs.

Reply:
Fixed